apps/mobile/src/components/Feed/Socialize/Comments.tsx

```diff
@@ -0,0 +1,53 @@
+import { View } from 'react-native';
+import { graphql, useFragment } from 'react-relay';
+
+import { GalleryTouchableOpacity } from '~/components/GalleryTouchableOpacity';
+import { Typography } from '~/components/Typography';
+import { CommentsFragment$key } from '~/generated/CommentsFragment.graphql';
+
+import { CommentLine } from './CommentLine';
+import { RemainingCommentCount } from './RemainingCommentCount';
+
+type Props = {
+  commentRefs: CommentsFragment$key;
+  totalComments: number;
+  onCommentPress: () => void;
+};
+
+export default function Comments({ commentRefs, totalComments, onCommentPress }: Props) {
+  const comments = useFragment(
+    graphql`
+      fragment CommentsFragment on Comment @relay(plural: true) {
+        dbid
+        ...CommentLineFragment
+      }
+    `,
+    commentRefs
+  );
```

Comment:
Since we fetch the comments inside this component, can we remove the `totalComments` props?

Reply:
unfortunately I don't think we can because this fragment is on the edge node, but the total comments comes from the pageInfo.total field 

that lets us get the total number of comments not just the number of comments loaded so far